### PR TITLE
Add global user status

### PR DESCRIPTION
**Pull request by SasLord**
<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjM5MDNjOWM2MTdkNTcxZjcwZTNhNTQiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.ECeH91QpPKm49NobIWFGe_ZYIMrssU2vClOva3jbglk">Huly&reg;: <b>UBERF-6818</b></a></sub>

### CODE REVIEW

**Comment by kristina-fefelova** [comment]
Added user status in different places:

- inbox
Screenshot from 2024-05-08 11-35-55

- attributes and lists
Screenshot from 2024-05-08 11-37-15

- navigator 
Screenshot from 2024-05-08 11-36-58

- issues assiggnes
Screenshot from 2024-05-08 11-36-49
- employees in different tables 
Screenshot from 2024-05-08 11-36-21

**Comment by aonnikov** [comment]
I think we should not add statuses everywhere. In activity they produce too much of visual noise when repeated in every avatar.

**Comment by kristina-fefelova** [comment]
> I think we should not add statuses everywhere. In activity they produce too much of visual noise when repeated in every avatar.

Do you mean inbox?
Yes, I can remove it in inbox

I didn't add status to activity

**Comment by aonnikov** [comment]
[Kristina Fefelova](https://front.hc.engineering/browse/?workspace=platform&_class=contact%3Aclass%3APerson&_id=63204fc031b9728f9e6b2e1a&label=Kristina%20Fefelova) yes, I mean inbox. Statuses make sense in places where we expect some immediate communication. When we select an assignee, for example, it is not needed.

File: plugins/contact-resources/src/components/AvatarInstance.svelte [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  export let standby: boolean = false
+  export let withStatus: boolean = false
+  export let element: HTMLElement
+
+  export function pulse (): void {
+    if (element) element.animate(pulsating, { duration: 150, easing: 'ease-out' })
+    if (standby) {
+      standbyMode = false
+      if (timer) clearTimeout(timer)
+      timer = setTimeout(() => {
```
Comment (haiodo) [inline plugins/contact-resources/src/components/AvatarInstance.svelte:37]: we should cancel timer on on destroy

**Review by haiodo — APPROVED** [review]

**Pull request merged**